packages/forms/resources/views/components/wizard/step.blade.php

```diff
@@ -12,6 +12,7 @@
 @endphp
 
 <div
+    x-bind:tabindex="$el.querySelector('[autofocus]') || '0'"
```

Comment:
This reads a little nicer.
Could you check if this still works fine (I would expect so), @mokhosh?

```suggestion
    x-bind:tabindex="$el.hasAttribute('autofocus') || '0'"
```

Reply:
No, it doesn't because it's not about this element having the `autofocus` attribute, it's about having any children that have `autofocus`.

Reply:
Obviously, I forgot 🤦‍♂️🤣